pkg/ccl/changefeedccl/kvevent/blocking_buffer.go

```diff
@@ -24,6 +23,13 @@ import (
 	"github.com/cockroachdb/errors"
 )
 
+type queue interface {
+	enqueue(e Event)
+	dequeue() (Event, bool)
+	empty() bool
+	purge()
+}
```

Comment:
Please drop this interface and use chunk queue directly; I only added this interface
when testing alternative implementations.